espresso-server/app/src/androidTest/java/io/appium/espressoserver/lib/handlers/OpenNotifications.kt

```diff
@@ -0,0 +1,28 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.appium.espressoserver.lib.handlers
+
+import io.appium.espressoserver.lib.helpers.InteractionHelper.getUiDevice
+import io.appium.espressoserver.lib.model.AppiumParams
+
+class OpenNotifications : RequestHandler<AppiumParams, Void?> {
+    override fun handleInternal(params: AppiumParams): Void? {
+        getUiDevice().openNotification()
+        return null
```

Comment:
We do not need to return the result?
I mean the openNotification returns true or false.
So, should we raise an error if the result was false?

Reply:
makes sense. I will raise a separate PR